examples/angular-app/package.json

```diff
@@ -27,7 +27,7 @@
     "@angular/cli": "~15.1.4",
     "@angular/compiler-cli": "^15.1.3",
     "@types/jasmine": "~4.0.0",
-    "cypress": "^12.5.1",
+    "cypress": "latest",
```

Comment:
Have you tested this? AFAIK this won't have an impact if the package.lock still has the previous version.